docs/source/en/_toctree.yml

```
@@ -4,7 +4,7 @@
   - local: quicktour
     title: Quicktour
   - local: stable_diffusion
-    title: Stable Diffusion
+    title: Improve DiffusionPipeline performance
```

```suggestion
    title: Effective and Efficient Stable Diffusion
```
IMO it's not just performance, it's also showing an efficient workflow of how to arrive at a good image.

Maybe a title like the above is better?

---

Nice, I like the alliteration :)